deploy/cluster-libchart/templates/_network.tpl

```diff
@@ -0,0 +1,14 @@
+{{/*
+Define component services
+*/}}
+{{- define "cluster-libchart.componentServices" }}
+services:
+  {{- if .Values.hostNetworkAccessible }}
+  - name: vpc
+    serviceType: NodePort
+  {{- end }}
+  {{- if .Values.publiclyAccessible }}
+  - name: public
+    serviceType: LoadBalancer
+  {{- end }}
```

Comment:
there is also an "Intranet" behavior, that uses LB for intra-network (intranet VPC accessible)

Reply:
check with @iziang

Reply:
I have confirmed with @iziang that for the service, annotations have been added for different cloud providers.